Header header
int16 faultCode
